py-polars/polars/internals/__init__.py

```diff
@@ -28,3 +28,34 @@
 )
 from .series import Series, wrap_s
 from .whenthen import when  # used in expr.clip()
+
+__all__ = [
```

Comment:
Why not include `argsort_by`, `date_range`, and `_deser_and_exec` in this list? These are now imported but not available through `__all__`.

Reply:
Only because `mypy` didn't complain about them, meaning they are not imported from anywhere else in the codebase. It might better to remove them as imports from the `__init__` than to add them to `__all__`. Thoughts?

Reply:
You're right, let's keep that for a different PR!